src/javascript/Edit/save/save.gql-mutation.js

```diff
@@ -24,7 +25,7 @@ export const SavePropertiesMutation = gql`
                     delete(language: $language)
                 }
                 removeMixins(mixins: $mixinsToDelete)
-                mutateWipInfo(wipInfo:$wipInfo)
+                mutateWipInfo(wipInfo:$wipInfo) @include(if: $shouldSetWip)
```

Comment:
We should not only prevent to save it but also not display the button to set the WIP on the content